news/2022-03-07-community-meetings-recap.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: "Community Meetings Recap & Scoring Survey"
+date: 2022-03-18 14:00:00 +0000
+---
+
+Check out what's been happening in the new community meetings, and fill out a new survey to let us know your thoughts on how scoreboards should be handled in the future!
+
+![](https://assets.ppy.sh/media/generic-header.png)
+
+If you're not in the know, we've been hosting [community meetings](/wiki/Community/osu!_community_meetings) since mid-September in 2021. Originally, these meetings were an Q&A-style panel featuring [peppy](https://osu.ppy.sh/users/2), [smoogipoo](https://osu.ppy.sh/users/1040328) and [Ephemeral](https://osu.ppy.sh/users/102335) as they tackled specific topics and open questions from the community at large, mostly regarding the future of the game in things like lazer.
```

Comment:
```suggestion
If you're not in the know, we've been hosting [community meetings](/wiki/Community/osu!_community_meetings) since mid-September in 2021. Originally, these meetings were a Q&A-style panel featuring [peppy](https://osu.ppy.sh/users/2), [smoogipoo](https://osu.ppy.sh/users/1040328) and [Ephemeral](https://osu.ppy.sh/users/102335) as they tackled specific topics and open questions from the community at large, mostly regarding the future of the game with things like lazer.
```